examples/README.md

```diff
@@ -0,0 +1,228 @@
+# Examples
+

```

Comment:
Explain at the beginning that users should already have a Kubernetes Master/Cluster running, provide example to https://github.com/cloudify-examples/simple-kubernetes-blueprint/tree/4.0.1